galsim_extra/mixed_scene.py

```diff
@@ -56,10 +58,35 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
         # Add objects field to the ignore list
         # Also ignore magnify and shear, which we allow here for convenience to act on whichever
         # object ends up being chosen.
-        ignore = ignore + ['objects', 'magnify', 'shear', 'obj_type']
+        ignore = ignore + ['objects', 'magnify', 'shear', 'obj_type', 'shear_scene']
+
+        stamp_xsize, stamp_ysize, image_pos, world_pos = super(MixedSceneBuilder, self).setup(config,base,xsize,ysize,ignore,logger)
+        
+        if 'shear_scene' in config:
+            shear_scene = galsim.config.ParseValue(config, 'shear_scene', base, bool)[0]
+        else:
+            shear_scene = False
+        
+        # option to shear the full scene.
+        if shear_scene:       
```

Comment:
We actually only want to shear the galaxies here so I think we want shear_scene to be a dictionary with True or False keyed on the object type.

Reply:
You can just let the user do that in the config file.  `shear_scene: "$@stamp.obj_type == 'gal'"`

Reply:
ahhhh